app/models/reservation.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Reservation < ApplicationRecord
+  belongs_to :user
+  belongs_to :reservation_status
```

Comment:
Do you have any more statuses? For clarity, you can consider creating a module for Reservation models, and then there would for example `Reservations::Reservation` and `Reservations::Status` and then no reservation_status column in the database. 

FYI this is just an idea.

Reply:
I am planning to have 3 statuses: cancelled, pending and paid. Do you think that a module would be here a better idea? If yes, why? Would it have any advantages over the separate table?

Reply:
1. In terms of the module I was thinking about namespace so 

```
module Reservations 
  class Reservation < ActiveRecord::Base
  ...
  end
end
```

and 
```
module Reservations
  class Status < ActiveRecord::Base
  ...
  end
end

Reply:
A separate status table is the same thing as a ticket_type but there's a higher chance that there would be a need for dynamic creation of these statuses but it's all debatable :)

Reply:
I removed the table, thanks!